README.md

```diff
@@ -32,74 +32,6 @@ This project contains numerous packages. See individual package docs for corresp
 | [home_robot_sim](src/home_robot_sim) | Simulation |
```

Comment:
i think we need to provide more positive guidance at the beginning of this section. like "to install simulation..." or "to start on hardware..."